docs/crop.md

```diff
@@ -1,67 +1,42 @@
-# Crop
+# **Crop**
 
-You can remove unwanted outer areas from an image by cropping an image. There are two ways with Pinta that you can crop an image:
+You can remove unwanted outer areas from an image by cropping the image. There are two ways with Pinta that you can crop an image:
 
-  - Crop to Selection: Use to remove part of the image surrounding the selection. The selected part of the image will exand to the size of the canvas. For irregulary shaped selection, such as those made by the Lasso too, the image is cropped to a rectangular bounding box that contains the selection).
-  - Auto Crop - Use this feature to remove small border areas from an image. For example sometimes you may have a photo with a white border that you would like to remove and this feature enables you to remove it.
+  - **Crop to Selection**: Use to remove part of the image surrounding the selection. The selected part of the image will expand to the size of the canvas. For irregular-shaped selection, such as those made by the **Lasso Select** tool, the image is cropped to a rectangular bounding box that contains the selection.
+  - **Auto Crop** - Use this feature to remove small border areas from an image. For example, sometimes you may have a photo with a white border that you would like to remove and this feature enables you to remove it.
 
 ## **Crop to Selection** 
 
-1. Select the image you want to crop from the *Images Window*.  
-
-     ![Image Window select image](img/crop/imageflowerselect.png)  
-
+1. Select the image you want to crop from the **Images** window.  
+  ![Image Window select image](img/crop/imageflowerselect.png)  
 
     &nbsp; 
- 
   
-2. [Select part of an image you want to crop using one of the selection tools](select_overview.md)
-
-The following example used the rectangle selection to select part of the image:
-
-![Crop Example](img/crop/cropexample.png)
-
-
-3. Go to Image Menu and Choose Crop to Selection.
-
-![Select crop](img/crop/croptoselectionwindow.png)
-
-
-Here is what the example image looks like after chossing **Crop to Selection**
-
-
-![Crop Example Final](img/crop/selectcrop.png)
+2. [Select part of an image you want to crop using one of the selection tools](select_overview.md).  
 
+The following example uses the rectangle selection to select part of the image:  
+  ![Crop Example](img/crop/cropexample.png)  
 
+3. Go to **Image > Crop to Selection**.  

```

Comment:
Need to Select Crop to Selection... so should say Go to **Image ** and select Crop to Selection